### ISSUE TITLE
Add guidance on PR approval patterns

### ISSUE THREAD

**Issue by malcolmbarrett** [StanfordHPDS/lab_manual#35]
I think we need to formalize the expectations here:

Request a review: request a reviewer in GitHub in the PR
Approve a review: Approved in PR mechanism
Approved with revisions: Approved in PR mechanism and verbally request changes prior to merging via comments
Request changes and continue the review after changes: Changes requested in PR mechanism and continue the review after changes made

**Comment by sherrirose** [comment]
Agree. I know I haven't always followed the same pattern for this either!

**Comment by sherrirose** [comment]
Another pattern: who merges?
